src/main/webapp/WEB-INF/appengine-web.xml

```diff
@@ -15,7 +15,7 @@
   limitations under the License.
 -->
 <appengine-web-app xmlns="http://appengine.google.com/ns/1.0">
-    <application>team33-197219</application>
+    <application>team-33-project-204314</application>
```

Comment:
Can we do a unique name here, not tied to team 33 and a unique identifier?

Reply:
I don't think the ID is up to us

Reply:
I think I was able to change the ID, will try to find where I did that...

Reply:
I think we can do that but we need `resourcemanager.projects.update` permissions

Reply:
Actually it seems that this ID was set when the project was created, and can't be changed past the creation

Reply:
I made it on the site, whatever it said to do in project 0, and I already added everyone with the right permissions.

Reply:
Sounds good